app/models/UserAnswers.scala

```diff
@@ -95,7 +98,8 @@ object UserAnswers {
     (
       (__ \ "_id").read[String] and
         (__ \ "data").read[JsObject] and
-        (__ \ "internalId").read[String]
+        (__ \ "internalId").read[String] and
+        (__ \ "is5mldEnabled").read[Boolean]
```

Comment:
We need to have `readWithDefault[Boolean](false)` here now as we uncovered on Friday this has been a bug causing lost data

Reply:
Updated